chart/hyrax/values.yaml

```diff
@@ -0,0 +1,97 @@
+# Default values for hyrax.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: samveralabs/dassie
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: hyrax.local
+      paths: []
+  tls: []
+
+resources: {}
+
+fcrepo:
+  enabled: true
+  fcrepoSecretName: "hyrax.fcrepo.fullname"
```

Comment:
having fcrepo and hyrax share a postgres deployment seems unusual for default behavior, imo

Reply:
it seems a lot better than deploying two separate postgresql instances for one application stack?

are you thinking we'll want to scale these independently, or???

Reply:
no it makes sense, but it took a while for me to see what was going on and it's not a pattern i've seen often

so maybe just a note in the readme or a comment here to clarify

Reply:
i don't really love the secret injection thing. ideally the hyrax chart wouldn't know so much about fedora's `JAVA_OPTS`, but i ran out of energy for perfecting it.